cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -442,7 +442,10 @@ pGovernanceCmd :: Parser GovernanceCmd
 pGovernanceCmd =
   Opt.subparser $
     mconcat
-      [ Opt.command "create-update-proposal"
+      [ Opt.command "create-MIR-certificate"
```

Comment:
I need to check if this is case-sensitive. If so, I think we should instead use:

```suggestion
      [ Opt.command "create-mir-certificate"
```

Reply:
It appears this is case-sensitive. Not sure what others think, but I'd prefer sticking with commands and options in all lowercase.